spyder/plugins/editor/widgets/editor.py

```diff
@@ -499,10 +500,21 @@ def __init__(self, parent, actions):
             _("Find symbols in file..."),
             icon=ima.icon('symbol_find'),
             triggered=self.open_symbolfinder_dlg)
-        copy_to_cb_action = create_action(self, _("Copy path to clipboard"),
-                icon=ima.icon('editcopy'),
-                triggered=lambda:
-                QApplication.clipboard().setText(self.get_current_filename()))
+        copy_absolute_path_action = create_action(
+            self,
+            _("Copy abosolute path"),
+            icon=ima.icon('editcopy'),
+            triggered=lambda:
+            QApplication.clipboard().setText(
+                self.get_current_filename()))

```

Comment:
Let's move this to a method called `copy_absolute_path`, to make things easier to read.